cloud-config/cloud-config-main.yml

```diff
@@ -116,6 +131,12 @@ vm_types:
 - name: logsearch_errand
   cloud_properties:
     instance_type: t2.medium
+- name: minion
```

Comment:
Prefix with `kubernetes_` for clarity. Also, won't we also need `etcd` and `consul` types?